app/build.gradle

```diff
@@ -74,71 +75,94 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile supportDependencies.appCompat
-    compile supportDependencies.design
-    compile supportDependencies.supportAnnotation
-    compile playServicesDependencies.maps
-    compile playServicesDependencies.analytics
+
+    compile libraries.support.appCompat
+    compile libraries.support.cardView
+    compile libraries.support.design
+    compile libraries.support.recyclerView
+    compile libraries.support.supportAnnotation
+    compile libraries.support.v4
+
+    compile libraries.playServices.all
```

Comment:
Why are you adding `all` and also so many of the play libs? Previously there were only `maps` and `analytics`